Makefile

```
@@ -123,6 +123,8 @@ playwright-examples: install-playwright ## end-to-end tests on examples with pla
 
 playwright-debug: install-playwright ## All end-to-end tests, chrome only, headed
 	pytest -c tests/playwright/playwright-pytest.ini tests/playwright/$(SUB_FILE)
+playwright-show-trace: ## Show trace of failed tests
+	npm playwright show-trace test-results/*/trace.zip
```

```suggestion

playwright-show-trace: ## Show trace of failed tests
	npm playwright show-trace test-results/*/trace.zip
```

---

Is this intended to be run by humans at the console? It seems like maybe not? In that case, lets take out the `## help comment`

```suggestion

playwright-show-trace:
	npm playwright show-trace test-results/*/trace.zip
```

---

It's intended to be run by humans

```suggestion

playwright-show-trace: ## Show trace of failed tests
	npx playwright show-trace test-results/*/trace.zip
```